changelog.d/9548.bugfix

```diff
@@ -0,0 +1 @@
+/register now requires a `body.type` value of `m.login.appservice` when registering appservice users.
```

Comment:
this should probably be a `removal` (see https://github.com/matrix-org/synapse/blob/develop/CONTRIBUTING.md#changelog), with slightly different wording.

Reply:
I've made an attempt, but I'm not sure what wording change you are looking for.